### ISSUE TITLE
Loading spinner before image loads

### ISSUE THREAD

**Issue by nkhaitan** [evgenyneu/Auk#17]
Hello,

Is there a way we could have a progress spinner show up before the image loads completely?

Thanks,

Nilesh

**Comment by evgenyneu** [comment]
Hi, @nkhaitan, no, unfortunately there is no spinner right now.

This will a good feature to add. Feel free to submit a pull request. It can be done by adding a new settings like this:

```Swift
scrollView.auk.settings.showSpinnerWhileLoading = true
```

I personally do not like spinners and prefer placeholder images instead.

// Show placeholder image while remote image is being downloaded.
scrollView.auk.settings.placeholderImage = UIImage(named: "placeholder.jpg")

You can see how it works and looks in the demo app.

**Comment by nkhaitan** [comment]
Thanks a lot man! Will look into it. Also, is there a way I can add a button on the bottom right of each image?

**Comment by evgenyneu** [comment]
@nkhaitan, yes, it can be done. Each page in the scroll view is an instance of [AukPage](https://github.com/evgenyneu/Auk/blob/master/Auk/AukPage.swift) class which is a UIView subclass. 

You can fork the library and add other subviews such as buttons into it. What we can is to expose a public property that contains that pages, something like this:

```Swift
let slideshowPages = scrollView.auk.pages // Array of UIView objects
slideshowPages[0].addSubview(button)
// Position button with Auto Layout on the bottom right.
```

This will make it easier for other people to add subviews to pages without a need to maintain their own fork of the library. Feel free to submit a pull request with the `pages` property.

**Comment by J-phantom** [comment]
Hello i use: `scrollView.auk.settings.showSpinnerWhileLoading = true `
but error `Value of type 'AukSettings' has no member 'showSpinnerWhileLoading'`
